### ISSUE TITLE
Theme. How to apply differents styles to differents kind of comments ?

### ISSUE THREAD

**Issue by FletcherPink** [adobe/brackets#10220]
Hello,
I discovered " Brackets" today and created a theme.
I will wish to apply a CSS style according to the kind of comment:

    /* Comment */ (style one)
    /** Comment **/ (style two)
    // Comment (style three)

Is there a way to do this?
Thank you

**Comment by redmunds** [comment]
@FletcherPink This is an interesting technique. To work on changes, you'll need to keep track of the current document and listen for the "change" event.

Marking as an `Extension Idea`. Since this is not a bug, I'm closing.

**Comment by MiguelCastillo** [comment]
@FletcherPink I can help you get this working if you start an extension you point me to it.

**Comment by peterflynn** [comment]
@MiguelCastillo may have this in mind already, but just as a note: doing this on every "change" event would probably make typing performance very poor.  The best option is probably to implement a CodeMirror overlay mode.  See [here](http://codemirror.net/demo/mustache.html) for an example and [here](https://groups.google.com/forum/#!topic/brackets-dev/cePUzc4oOTs) for discussion of applying overlay modes within Brackets.

**Comment by peterflynn** [comment]
Another way to approach this btw would be a pull request to CodeMirror to apply such distinguishing classes for everyone.  This would probably be very helpful in other places too (e.g. it might significantly simplify the comment-toggling code in our EditorCommandHandlers module).

**Comment by FletcherPink** [comment]
@peterflynn That's exactly what I thought. CodeMirror seems to be the source to work. I tried to understand how CodeMirror works but my knowledge is too low so I gave up . Have you some tips and some explanation to tell me ?

@MiguelCastillo Thank you for your proposal . As an amateur , I will be honored to perform extension partenanriat with an expert. I have much to learn. I will focus on the overlay parsing and as soon as I wrote something , I will share it here .
